### PR TITLE
doc: add JetBrains VMOptions config

### PR DESCRIPTION
**Pull request by yaohui-wyh**
## Description

Add doc about JetBrains VMOptions config. Feature implemented at https://github.com/gitpod-io/gitpod/pull/10768. cc @akosyakov 

## Related Issue(s)

https://github.com/gitpod-io/gitpod/issues/8704

## Release Notes

```release-note
Add doc about JetBrains VMOptions config.
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2275"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2275"></a>

### CODE REVIEW

File: gitpod/docs/references/gitpod-yml.md [hunk 1]
```diff
@@ -383,6 +384,28 @@ The `version` is defined as follows:
 
 </div>
 
+### `jetbrains.[product].vmoptions`
+
+> JetBrains VM options support (via gitpod.yml) is currently in [Alpha](/docs/references/gitpod-releases) · [Send feedback](https://github.com/gitpod-io/gitpod/issues/8704).
```
Comment (akosyakov) [inline gitpod/docs/references/gitpod-yml.md:389]: Configuration of JVM options (via gitpod.yml) is currently in..

**Review by akosyakov — COMMENTED** [review]
thank you, see my comments

File: generators/ide/jb-product.md [hunk 2]
```diff
@@ -145,6 +145,20 @@ jetbrains:
...
+
+You can adjust VM options for the IDE backend server, especially if you want to increase the `-Xmx` memory size. For example:
+
+```yaml
+jetbrains:
+  __productId__:
+    vmoptions: "-Xmx4g"
+```
+
+For more detailed information on VM options, refer to [Common JVM Options](https://www.jetbrains.com/help/idea/tuning-the-ide.html#common-jvm-options) from JetBrains documentation.
```
[the remark this answers is not in the document]
Comment (yaohui-wyh) [reply generators/ide/jb-product.md:160]: Sadly this does not work for GoLand, as the URl slug for GoLand is `go`:
https://www.jetbrains.com/help/go/tuning-the-ide.html#configure-jvm-options

[the remark this answers is not in the document]
Comment (loujaybee) [reply generators/ide/jb-product.md:160]: If necessary, we can add a [redirect](https://github.com/gitpod-io/website/blob/main/_redirects) / update the slug.

[the remark this answers is not in the document]
Comment (yaohui-wyh) [reply generators/ide/jb-product.md:160]: > If necessary, we can add a [redirect](https://github.com/gitpod-io/website/blob/main/_redirects) / update the slug.

Thanks! I will try it

**Pull request merged**